src/translator/service.h

```diff
@@ -83,9 +84,11 @@ class Service {
   /// @param [in] shortlistMemory byte array of shortlist (aligned to 64)
   explicit Service(const std::string &config,
                    AlignedMemory modelMemory = AlignedMemory(),
-                   AlignedMemory shortlistMemory = AlignedMemory())
+
+                   AlignedMemory shortlistMemory = AlignedMemory(),
+                   const std::vector<string_view> *vocabsMemoryView = nullptr)
```

Comment:
Can it be `const std::vector<AlignedMemory> vocabMemory`? It would be consistent with rest of the 2 arguments 👍

Reply:
Yes. This is fixed.